scripts/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "license": "MIT",
+  "scripts": {
+    "test": "jest --ci --reporters=default --testResultsProcessor=jest-junit ${TEST_PATTERN}"
```

Comment:
the TEST_PATTERN should just be passed as an argument on the command line no need for an env var here